src/askui/tools/askui/command_helpers.py

```diff
@@ -0,0 +1,200 @@
+from pydantic import Field, constr
+from typing_extensions import Annotated, Union
+
+from askui.tools.agent_os import Coordinate
+from askui.tools.askui.askui_ui_controller_grpc.generated.AgentOS_Send_Request_2501 import (  # noqa: E501
+    AskuiAgentosSendRequestSchema,
+    Command,
+    Command1,
+    Command2,
+    Command3,
+    Command4,
+    Command5,
+    Guid,
+    Header,
+    Length,
+    Location2,
+    Message,
+    Name,
+    Name1,
+    Name2,
+    Name3,
+    Name4,
+    Name5,
+    RenderImage,
+    RenderLinePoints,
+    RenderObjectId,
+    RenderObjectStyle,
+    RenderText,
+)
+
+LengthType = Union[
+    Annotated[str, constr(pattern=r"^(\d+(\.\d+)?(px|%)|auto)$")], int, float
+]
+
+ColorType = Union[
+    Annotated[str, constr(pattern=r"^#([0-9a-fA-F]{6})$")],  # Hex color like #RRGGBB
+    Annotated[
+        str, constr(pattern=r"^rgb\(\s*(\d{1,3})\s*,\s*(\d{1,3})\s*,\s*(\d{1,3})\s*\)$")
+    ],  # RGB color like rgb(255, 0, 128)
+]
+
+
+def create_style(
+    top: LengthType | None = None,
+    left: LengthType | None = None,
+    bottom: LengthType | None = None,
+    right: LengthType | None = None,
+    width: LengthType | None = None,
+    height: LengthType | None = None,
+    color: ColorType | None = None,
+    opacity: Annotated[float, Field(ge=0.0, le=1.0)] | None = None,
+    visible: bool | None = None,
+    font_size: LengthType | None = None,
+    line_width: LengthType | None = None,
+) -> RenderObjectStyle:
+    """Create a style object with the specified properties."""
+
+    return RenderObjectStyle.model_validate(
+        {
+            "top": top,
+            "left": left,
+            "bottom": bottom,
+            "right": right,
+            "width": width,
+            "height": height,
+            "color": color,
+            "opacity": opacity,
+            "visible": visible,
+            "font-size": font_size,
+            "line-width": line_width,
+        }
+    )
+
+
+def create_get_mouse_position_command(
+    session_guid: str,
+) -> AskuiAgentosSendRequestSchema:
+    command = Command(name=Name.GetMousePosition)
+
+    header = Header(authentication=Guid(root=session_guid))
+    message = Message(header=header, command=command)
+    return AskuiAgentosSendRequestSchema(message=message)
+
+
+def create_set_mouse_position_command(
+    x: int, y: int, session_guid: str
+) -> AskuiAgentosSendRequestSchema:
+    location = Location2(x=Length(root=x), y=Length(root=y))
+    command = Command1(name=Name1.SetMousePosition, parameters=[location])
+
+    header = Header(authentication=Guid(root=session_guid))
+    message = Message(header=header, command=command)
+    return AskuiAgentosSendRequestSchema(message=message)
```

Comment:
I think this is a nice place for an `AskuiAgentosSendRequestSchemaBuilder` (class) (builder pattern) that is initialized with the session guid. Could remove a lot of code duplication where each builder method can focus on the actual command. But I think I would just leave it for now as it is :)